README.md

```diff
@@ -43,6 +43,24 @@ const manager = fileManagement.create('S3', {
       .catch(console.error);
 ```
 
+#### Run cloudfront invalidation
+After `upload` function is executed, it exposes `invalidate` function along with the original upload result:
+```js
+manager.uploadFile(...args)
+.then(({ invalidate, result }) => {
```

Comment:
this is still a breaking change for this version, we then need a new major version or not put invalidation here at all

Reply:
I forgot to update readme. it does not work like this anymore